config.ts

```diff
@@ -1,4 +1,11 @@
+const PIECE_RESOURCE_BASE = 'http://themis.vlaanderen.be/id/stuk/';
+const DOCUMENT_CONTAINER_RESOURCE_BASE = 'http://themis.vlaanderen.be/id/serie/';
 const FILE_RESOURCE_BASE = 'http://themis.vlaanderen.be/id/bestand/';
+
+const VERTROUWELIJK = 'http://themis.vlaanderen.be/id/concept/toegangsniveau/9692ba4f-f59b-422b-9402-fcbd30a46d17';
+const INTERN_OVERHEID = 'http://themis.vlaanderen.be/id/concept/toegangsniveau/634f438e-0d62-4ae4-923a-b63460f6bc46';
+const BESLISSINGSFICHE_TYPE = 'http://themis.vlaanderen.be/id/concept/document-type/e807feec-1958-46cf-a558-3379b5add49e';
```

Comment:
`VERTROUWELIJK` no longer used. Cleanup can be done
`BESLISSINGSFICHE_TYPE` currently used, but don't think it should be. Bundled pdf's have no doc-type.